shelley/design-spec/delegation_design_spec.tex

```diff
@@ -2965,31 +3025,75 @@ \subsubsection{Non-Myopic Pool Stake}
+performance \(\pbar\), for a member contributing member stake \(t\), are
 \begin{equation}
-    r_\mathrm{member, nm}(c, m, s, \sigma, t, r) :=
-    r_\mathrm{member}\Bigl(\hat{f}\bigl(s,\sigma_\mathrm{nm}(s,\sigma, r)\bigr), c, m, t, \sigma_\mathrm{nm}(s,\sigma,r)\Bigr).
+    r_\mathrm{member, nm}(c, m, s, \sigma, t, r, \pbar) :=
+    r_\mathrm{member}\Bigl(\hat{f}\bigl(s,\sigma_\mathrm{nm}(s,\sigma, r),
+    \pbar\bigr),
+    c, m, t, \sigma_\mathrm{nm}(s,\sigma,r)\Bigr).
 \label{eq:non-myopic-member-rewards}
 \end{equation}
 This formula holds if the pool is amongst the top \(k\) rated pools. If it is
 not, we should expect no rational player to delegate to it, and accordingly we
 will have to replace \(\sigma_\mathrm{nm}\) by \(s + t\) in
 \cref{eq:non-myopic-member-rewards} -- the pool's stake will be entirely made up
-of the leader's stake and the stake that this member chooses to delegate.
+of the owners' stake and the stake that this member chooses to delegate.
+
+\subsubsection{Average Apparent Performance}
+
+Using the apparent performance of a pool \emph{within the last epoch} is not
+suitable for determining the long-term expected rewards for delegating to a
+pool. Rather, one should use the average apparent performance over several
+epochs. This avoids prefering or discarding a pool just because it performed
+exceptionally well or bad in one particular epoch. If the ratio of the number of
+stake pools and the number of expected blocks per epoch is large, this becomes
+even more important, since the apparent performance in a single epoch would be
+bound to fluctuate quite a bit.
+
+In order to decrease the influence of these fluctuations, and to get a long-term
+picture, when evaluating the desirability and non-myopic rewards, one should
+insert the apparent performance averaged over several epochs as \(\pbar\), not
+the value of \(\pbar\) from just the last epoch.
+
+Averaging can be done efficiently via an exponential moving average, or by using
+a moving window and keeping track of the apparent performance during the last
+couple of epochs. Note that since we perform averaging only when predicting the
+rewards to allow stakeholders to make an informed decision, but not when
+actually dealing out the rewards, it will be fine if different wallets will
+choose different methods of calculating the averages, since it does not affect
+the consensus.
+
+\subsubsection{Apparent Performance of New Pools}
+\label{apparent-performance-of-new-pools}
+
+When a new pool is created, there will be no data to determine its apparent
+performance, which is needed to calculate its desirability and non-myopic
+rewards. In order to include a new pool in the ranked display of pools, a wallet
+will have to pick a default value for the apparent performance.
+
+In order to neither advantage nor disadvantage new pools, we recommend
+initialising the moving average by looking at the top \(k\) stake pools (i.e.,
```

Comment:
How do we initialize the apparent performance for the very first pools?

Reply:
Right, we need to bootstrap 🤔 

I think it's fine if we just set the performance of the pools that join during the first couple of epochs (maybe while the decentralisation switch is very close to centralised, where it's harder to measure the performance) to `1`.